### DISCUSSION TITLE
Application Bug- My school does not have an educational email address

### DISCUSSION THREAD

**Discussion by ChewyJah** [community/community#68131] [GitHub Education]
### Hi everyone,

Github insists on a school email address when there is no such thing. Can someone please change the settings so my application can go through.  School is : Mages Institute of Excellence.

**Comment by PavelS05** [comment]
Same problem here, did you find any solution?

**Reply by rayhan97794** [reply]
what problem

**Reply by ChewyJah** [reply]
Yeah, had to ask the school to give me a email, it was only available upon request, but you Github Staff said if the school stops providing emails, you need to ask the school to write a letter stating that they no longer provide student emails.
